runtimes/common/src/migrations.rs

```diff
@@ -15,3 +15,55 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use sp_io::MultiRemovalResults;
+use sp_std::marker::PhantomData;
+
+use frame_support::{
+	traits::{Get, OnRuntimeUpgrade},
+	StorageHasher, Twox128,
+};
+
+pub struct RemoveRelocationPallets<R>(PhantomData<R>);
+
+impl<R: frame_system::Config> OnRuntimeUpgrade for RemoveRelocationPallets<R> {
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<(), &'static str> {
+		log::info!("Pre check RemoveRelocationPallets.");
+		let has_storage = frame_support::storage::migration::have_storage_value(
+			b"RelayMigration",
+			b"RelayNumberStrictlyIncreases",
+			b"",
+		) && frame_support::storage::migration::have_storage_value(b"DynFilter", b"Filter", b"");
+		if has_storage {
+			Ok(())
+		} else {
+			Err("Pallets not present")
+		}
+	}
+
+	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+		let entries: u32 = 1;
+		// there is no sensible action in case of a failure. We will check manually
+		// after the migration was done.
+		let _ = frame_support::storage::unhashed::clear_prefix(&Twox128::hash(b"RelayMigration"), Some(entries), None);
+		let _ = frame_support::storage::unhashed::clear_prefix(&Twox128::hash(b"DynFilter"), Some(entries), None);
```

Comment:
S: Add least post some success and debug message based on the result of the clearing.